lib/index.d.ts

```diff
@@ -44,6 +44,13 @@ export type InputMetadata = {
   }
 };
 
+export type ArchiveHeader = {
+  // The JSON parsed header string
+  header: any;
```

Comment:
Hmm. Seems like this should be a bit more strict than `any`, since we control what the header looks like.

Reply:
Fair enough, added the type info 👍